### PR TITLE
Pango update to 1.47, use clang linker

### PR DESCRIPTION
**Pull request by satmandu**
Part of upcoming adwaita PR.

Works properly:
- [x] x86_64

### CODE REVIEW

File: packages/pango.rb [hunk 1]
```diff
@@ -3,34 +3,30 @@
...
   depends_on 'libxrender'
-  depends_on 'fribidi'
+  depends_on 'fribidi' # Gets built inside install automatically.
   depends_on 'six'
+  depends_on 'llvm' => ':build'
 
   def self.build
-    system "meson --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} builddir"
+    ENV['CFLAGS'] = "-fuse-ld=lld"
+    ENV['CXXFLAGS'] = "-fuse-ld=lld"
```
Comment (uberhacker) [inline packages/pango.rb:24]: Isn't this going to break arm builds?  That has been my experience with all other packages that included these options.

Comment (satmandu) [reply packages/pango.rb:24]: I'm happy to try these on an arm build. What is the target machine so I can pull down the recovery image and test?

Comment (uberhacker) [reply packages/pango.rb:24]: Target would be `armv7l`.

File: packages/pango.rb [hunk 2]
```diff
@@ -3,34 +3,30 @@
...
+    system "meson",
+      "-Dprefix=#{CREW_PREFIX}",
+      "-Dlibdir=#{CREW_LIB_PREFIX}",
```
Comment (uberhacker) [inline packages/pango.rb:27]: Can we use the new `CREW_MESON_OPTIONS` here?

Comment (satmandu) [reply packages/pango.rb]: done

**Comment by satmandu** [comment]
FYI this needs the new harfbuzz in https://github.com/skycocker/chromebrew/pull/4456 .

**Pull request merged**